CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 ## Usage of StarcounterClientFiles 2.x in Level1 build automation
 
-Whatever is pushed on branch `2.x`, will be bundled in the Starcounter installer by the Level1 build automation. See [the announcement](https://github.com/Starcounter/AdminTrack/issues/438) for more details. 
+Whatever is pushed on branch `2.x`, in NOT bundled in the Starcounter installer by the Level1 build automation.
```

Comment:
I assume you mean "**is** not".

Anyways I don't see the need for "whenever" any more. "Whenever X happens Y won't happen". Doesn't really follow. 

I would go for "Branch `2.x` is NOT bundled in the Starcounter installer by the Level1 build automation."